src/content/docs/ai-observability/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Introduction to AI Monitoring'
+metaDescription: 'AI Monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import apmAiMonitoringAllCapabilities from 'images/apm_screenshot-full_ai-monitoring-all-capabilities.gif'
+
+Your AI-powered app introduces new technologies into your environment that are hard to monitor, let alone understand. With AI Monitoring, you can bridge unfamiliar technologies to familiar solutions, deepening your knowledge about your AI's behavior so you can improve its performance. 
```

Comment:
It might be good to define a little more clearly what we mean specifically by "AI". It's a term that's pretty muddy right now. I think you'll establish credibility right away by getting technically specific right out of the gate.